# SPDX-License-Identifier: MIT
# SPDX-FileCopyrightText: pooyanjamshidi

float64 charge
---
bool result